contracts/chain-adapters/Arbitrum_CustomGasToken_Funder.sol

```diff
@@ -0,0 +1,28 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+
+contract Arbitrum_CustomGasToken_Funder is Ownable {
+    using SafeERC20 for IERC20;
+
+    /**
+     * @notice Deposit tokens into the contract.
+     * @param token Token to deposit.
+     * @param amount Amount of tokens to deposit.
+     */
+    function deposit(IERC20 token, uint256 amount) external {
+        token.safeTransferFrom(msg.sender, address(this), amount);
+    }
```

Comment:
nit: do we need a deposit function if we can just transfer tokens in?

Reply:
true, will remove